File: toDoApp.java
// ToDoApp.java
import java.util.*;

public class toDoApp {
    static ArrayList<String> tasks = new ArrayList<String>();

    public static void addtask(String t) {
        tasks.add(t);
        System.out.println("Task Added!!");
    }

    public static void showTasks() {
        if (tasks.isEmpty()) {
            System.out.println("No tasks yet");
        } else {
            for (int i = 0; i < tasks.size(); i++) {
                System.out.println((i + 1) + ". " + tasks.get(i));
            }
        }
    }

    public static void removeTask(int n) {
        if (n > 0 && n <= tasks.size()) {
            tasks.remove(n - 1); 
            System.out.println("Task removed!");
        } else {
            System.out.println("Invalid task number!");
        }
    }

    public static void main(String args[]) {
        Scanner s = new Scanner(System.in);
        while (true) {
            System.out.println("--------------------------"); // para malupit ung format man
            System.out.println("1 Add Task"); 
            System.out.println("2.Show Tasks");
            System.out.println("3.Remove Task");
            System.out.println("4 Exit");
            System.out.println("--------------------------");
            System.out.print("Enter Choice: ");

            int choice = s.nextInt(); 

            switch (choice) {
                case 1:
                    System.out.print("Enter Task: ");
                    s.nextLine();
                    String t = s.nextLine();
                    addtask(t);
                    break;
                case 2:
                    showTasks();
                    break;
                case 3:
                    System.out.print("Enter task no to remove: ");
                    int n = s.nextInt();
                    removeTask(n); 
                    break;
                case 4:
                    s.close();
                    break;
                default:
                    System.out.print("Wrong Choice!!");
                    break;
            }
            if (choice == 4) {
                break;
            }
        }
    }
}

File: toDoApp.py
# toDoApp.py

tasks = []

def add_task(task):
    tasks.append(task)
    print("task added!")

def show_tasks():
    if len(tasks) == 0:
        print("no tasks yet")
    else:
        for i in range(len(tasks)):
            print(i + 1, ".", tasks[i])

def remove_task(tasknumber):
    tasks.pop(tasknumber)
    print("task removed!!")

def main():
    while True:
        print("1 Add Task")
        print("2.Show Tasks")
        print("3.Remove Task")
        print("4. Exit")
        ch = input("Enter choice : ")
        
        if ch == "1":
            t = input("Enter task : ")
            addtask(t)
        elif ch == "2":
            showTasks()
        elif ch == "3":
            n = int(input("Enter task number to remove: "))
            removetask(n)
        elif ch == "4":
            break
        else:
            print("Wrong choice!")

if __name__ == "__main__":
    main()
